### PR TITLE
Volume expand fails to start bricks when volume is in started state

### PR DESCRIPTION
**Pull request by vpandey-RH**
Fixes [#1329](https://github.com/gluster/glusterd2/issues/1329)

Volume expand fails to start bricks when volume is in started state but works fine when volume is in Stopped/Created state

Fix - Brick Volfiles need to be generated beafore starting the bricks.

### CODE REVIEW

**Comment by aravindavk** [comment]
add to whitelist

File: glusterd2/commands/volumes/volume-expand.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ func registerVolExpandStepFuncs() {
 		{"vol-expand.ValidateBricks", validateBricks},
 		{"vol-expand.InitBricks", initBricks},
 		{"vol-expand.UndoInitBricks", undoInitBricks},
+		{"vol-expand.GenerateBrickVolfiles", txnGenerateBrickVolfiles},
+		{"vol-expand.GenerateBrickvolfiles.Undo", txnDeleteBrickVolfiles},
```
Comment (rishubhjain) [inline glusterd2/commands/volumes/volume-expand.go]: Please rename "GenerateBrickvolfiles" as above.

File: glusterd2/commands/volumes/volume-expand.go [hunk 2]
```diff
@@ -134,6 +136,12 @@ func volumeExpandHandler(w http.ResponseWriter, r *http.Request) {
 			DoFunc: "vol-expand.UpdateVolinfo",
 			Nodes:  []uuid.UUID{gdctx.MyUUID},
 		},
+		{
+			DoFunc:   "vol-expand.GenerateBrickVolfiles",
+			UndoFunc: "vol-expand.GenerateBrickvolfiles.Undo",
```
Comment (rishubhjain) [inline glusterd2/commands/volumes/volume-expand.go]: Same as above.

File: glusterd2/commands/volumes/volume-expand.go (hunk 1 above)
Comment (vpandey-RH) [reply glusterd2/commands/volumes/volume-expand.go]: Done

File: glusterd2/commands/volumes/volume-expand.go (hunk 2 above)
Comment (vpandey-RH) [reply glusterd2/commands/volumes/volume-expand.go]: Done

**Comment by vpandey-RH** [comment]
Can we get this merged ?

**Pull request merged**